npm/publishing.md

```diff
@@ -0,0 +1,35 @@
+# Publishing to NPM
+
+The [Water Abstraction Helpers](https://github.com/DEFRA/water-abstraction-helpers) repo is publsihed to npm: https://www.npmjs.com/package/@envage/water-abstraction-helpers.
+
+
+To publsih a pakcgae to npm:
+- Create a branch to make changes
```

Comment:
Maybe add a step to say how the package-json version needs to be manually update ?

Reply:
I'd add this. everything else is so whizzy you might assume this is going to be done for you.

Reply:
```suggestion
- Create a branch to make changes
- Manually Update the package.json version number (This will be the tag version aswell)
```